pubspec.yaml

```diff
@@ -1,30 +1,30 @@
 name: flutter_launcher_icons
 description: A package which simplifies the task of updating your Flutter app's launcher icon.
-version: 0.11.0
+version: 0.11.0+1
 maintainer: Mark O'Sullivan (@MarkOSullivan94)
 homepage: https://github.com/fluttercommunity/flutter_launcher_icons
 repository: https://github.com/fluttercommunity/flutter_launcher_icons/
 issue_tracker: https://github.com/fluttercommunity/flutter_launcher_icons/issues
 
 dependencies:
-  args: ^2.2.0
-  checked_yaml: ^2.0.1
+  args: ^2.3.2
+  checked_yaml: ^2.0.2
   cli_util: ^0.3.5
-  image: ^3.0.2
-  json_annotation: ^4.5.0
-  path: ^1.8.0
-  yaml: ^3.1.0
+  image: ^4.0.10
```

Comment:
4.0.11 is available

Reply:
4.0.13 is available 😄

Reply:
I'm trying to update "image" package. But, I can't because "flutter_launcher_icons".

It would be awesome if this update was finished and merged.